notebooks/data-sources/oc-github-repo/github_PR_EDA.ipynb

```diff
@@ -2,7 +2,7 @@
  "cells": [
```

Comment:
<p>If you are performing the same linear operation on all of your data points (dividing by some value) I would not expect the distribution to change other than in the absolute values.</p><p> </p><p>To Identify they appropriate granularity of analysis I would think that you would want to tune the number of buckets, where the width of the bucket tells you the proper granularity.    </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/ocp-ci-analysis/pull/287/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='32'/>

Reply:
We have not explored this IIUC and we can look into this :+1: